### ISSUE TITLE
graph seems to not agree with soccoro data

### ISSUE THREAD

**Issue by wsmwk** [ashughes1/bugzilla-socorro-lens#17]
https://bugzilla.mozilla.org/show_bug.cgi?id=824713 bugzilla socorro lense graph (attached) doesn't agree with what socorro is telling me for the past month https://crash-stats.mozilla.com/signature/?signature=nsMsgDBView%3A%3AGetMsgHdrForViewIndex&date=%3E%3D2016-10-13T20%3A14%3A03.000Z&date=%3C2016-11-13T20%3A14%3A03.000Z#summary
![bug 824713 bugzilla inline crash graph](https://cloud.githubusercontent.com/assets/216903/20367638/6290a5dc-ac1e-11e6-9a6c-dfc2f07a36b4.png)

**Comment by ashughes1** [comment]
I took a quick look at this specific case and I believe the root cause is the same as issue #14. The add-on searches not just for the signatures listed in the bug but also for similar signatures. In this example, _nsMsgDBView::GetMsgHdrForViewIndex_ has 3 crashes for the last week but we also find _InvalidArrayIndex_CRASH | nsTArray_Impl<T>::ElementAt | nsMsgDBView::GetMsgHdrForViewIndex_ which has 51 crashes for the week.

At the API level I am searching for signature=~<something> instead of signature=<something> to capture crashes that may spread across other signatures not listed in the bug report. The downside of this approach would be that some crashes are over-reported in some cases. I could certainly change the add-on to count only those signatures explicitly listed in the bug report but that carries the risk of under-reporting in some cases.

So basically this comes down to a design decision and having to choose between the risk of over-reporting vs the risk of under-reporting. I made the decision to err on the side of over-reporting because I feel that's the safer bet for Firefox/Thunderbird/Fennec users. However I am open to debate on this decision.

Alternatively I could see about adding a toggle to the chart interface that allows the user decide.
